core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -652,6 +652,38 @@ $(document).ready(function () {
             executorSummaryTableSelector.column(14).visible(dataToShow.showBytesSpilledData);
           });
 
+        // Prepare data for speculation metrics
+        $("#speculationSummaryTitle").hide()
```

Comment:
nit: Some statements added by this change doesn't end with `;`.

Reply:
updated it